test/frame_rendering_test.dart

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('vm')
+import 'dart:async';
+import 'dart:io';
+
+import 'package:devtools/globals.dart';
+import 'package:devtools/service_extensions.dart' as extensions;
+import 'package:devtools/service_manager.dart';
+import 'package:devtools/timeline/frame_rendering.dart';
+import 'package:devtools/vm_service_wrapper.dart';
+import 'package:test/test.dart';
+
+import 'support/flutter_test_driver.dart';
+
+void main() {
+  group('frame rendering tests', () {
+    FlutterRunTestDriver _flutter;
+    VmServiceWrapper service;
+    FramesTracker framesTracker;
+
+    setUp(() async {
+      _flutter = FlutterRunTestDriver(Directory('test/fixtures/flutter_app'));
```

Comment:
I would steal the pattern from InspectorServiceTest that creates the app once and does minimal cleanup between tests.
Otherwise these tests take way too long to run as we are tearing down and setting up flutter envs a lot.
Bonus points: expose is as a shared helper library to build a flutter a flutter library and use it everywhere.

Reply:
Done.